python/astro_metadata_translator/translator.py

```diff
@@ -87,6 +88,57 @@ class MetadataTranslator:
     supported_instrument = None
     """Name of instrument understood by this translation class."""
 
+    @classmethod
+    def defined_in_class(cls, name):
+        """Report if the specified method name is defined in this class
+        or in a base class.
```

Comment:
I got tripped up with the wording here.  You mean "defined in this class specifically, or instead is defined in one of the classes this class inherits from".